### ISSUE TITLE
Struggling to use `GridLoad_mnc` or `GridLoad_mdsio` with my configuration

### ISSUE THREAD

**Issue by gaelforget** [gaelforget/MITgcmTools.jl#49]
This thread follows up on a question I received over email, which seems more useful to try address here in case others may have s similar issue.

Question was along the lines of:

"I went through the documentation of MITgcmTools.jl and MeshArrays.jl before asking, but I still can’t figure this out; the notebooks don’t really show how to load a setup that was not run using the julia wrapper setup. 

Specifically, I am trying to figure out how to create a gcmgrid from a netcdf grid file (of files). The GridLoad_mnc function from MITgcmTools.jl needs either a gcmgrid as input or a MITgcm_config, which is created when running the MITgcm using your tools."

**Comment by gaelforget** [comment]
@renskegelderloos, would you be able to provide feedback on this maybe?

**Comment by renskegelderloos** [comment]
These are all helpful. I think the biggest issue is when having a setup that has already been preprocessed and no longer adheres to the standard MITgcm tree. Also, the new GridLoad_mnc still assumes that there is a directory called "mnc_test_0001"; the MITgcm takes a user-defined variable "mnc_outdir_str" in data.mnc, so this can be different.

In non-standard cases, it would be really helpful to have a clear example of how to define a gcmgrid for cases that do not fall into one of the pre-defined gcm grid cases. I found a class 'll' (which I assumes stands for lat-lon) after digging deep, but it would be helpful to have that kind of class in a standard example. Once a gcmgrid is created, loading the grid with the examples provided is doable using the MITgcmTools.NetCDF.open function. 

For reference, this is a gcmgrid I created for a simple lat-lon setup: 

path  = "path/to/simulation/"
class = "ll"
nFaces= 1
np    = 960
nq    = 880

fSize = Array{NTuple{2, Int},1}(undef,nFaces)
fSize[:].=[(np,nq)]

ioSize = [np nq]
ioPrec = Float64

γ=gcmgrid(path,class,1,fSize, ioSize, ioPrec, read, write)
